example/lib/pages/buttons_page.dart

```diff
@@ -61,6 +61,13 @@ class _ButtonsPageState extends State<ButtonsPage> {
         ],
       ),
       children: [
+        const ResizablePane.noScrollBar(
+          minSize: 180,
+          startSize: 200,
+          windowBreakpoint: 700,
+          resizableSide: ResizableSide.right,
+          child: Center(child: Text('Resizable Pane')),
+        ),
```

Comment:
Can we actually put this in the new `ResizablePanePage` please?

Reply:
Just update one of the existing panes on that screen to be non-scrollable

Reply:
Yeah, sure !

Reply:
sure.